File: src/main/java/com/ninja/ieumnori/ieumnoriserver/inquiry/domain/Inquiry.java
package com.ninja.ieumnori.ieumnoriserver.inquiry.domain;

import com.ninja.ieumnori.ieumnoriserver.common.domain.BaseEntity;
import jakarta.persistence.*;
import lombok.*;

@Getter
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@AllArgsConstructor
@Builder
@Entity
@Table(name = "inquiry")
public class Inquiry extends BaseEntity {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "user_name", nullable = false)
    private String userName;

    @Column(name = "email", nullable = false)
    private String email;

    @Column(name = "title", nullable = false)
    private String title;

    @Lob
    @Column(name = "content", nullable = false)
    private String content;
}

File: src/main/java/com/ninja/ieumnori/ieumnoriserver/inquiry/domain/Inquiry.kt
package com.ninja.ieumnori.ieumnoriserver.inquiry.domain

import com.ninja.ieumnori.ieumnoriserver.common.domain.BaseEntity
import jakarta.persistence.*

@Entity
@Table(name = "inquiry")
class Inquiry (
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Long = 0,
    @Column(name="user_id", nullable = false)
    var userId: Long,
    @Column(nullable = false)
    var content: String,
    ): BaseEntity() {
    constructor(): this(0, 0, "")
}